pages/partials/questionAwardedPoints.ejs

```diff
@@ -1,11 +1,9 @@
 <% include('pointsFormatter'); %>
 
-<% for (var i = 0; i < awarded_points.length; i++) { %>
-    <% if (awarded_points[i] < init_points) { %>
-        <% if (awarded_points[i] > 0) { %>
-            <span class="badge badge-warning"><%= getStringFromFloat(awarded_points[i]) %></span>
-        <% } %>
-    <% } else { %>
-        <span class="badge badge-success"><%= getStringFromFloat(awarded_points[i]) %></span>
+<% if (previous_variants) { %>
+    <% for (var i = 0; i < previous_variants.length; i++) { %>
+        <span class="badge badge-secondary">
+            <a class="text-white" href="<%= urlPrefix %>/instance_question/<%= instance_question_id %>/?variant_id=<%= previous_variants[i].variant_id %>"><%= Math.round(previous_variants[i].max_submission_score * 100) + "%"  %></a> 
+        </span>
```

Comment:
As we discussed, I'm a bit worried about what happens when a student builds up a lot of previous variants. Here's an attempt to fix this by only showing the most recent 10 variants and providing a "..." clickable link that will reveal the earlier ones. Thoughts?

<img width="1120" alt="image" src="https://user-images.githubusercontent.com/875833/232249251-91bf1614-80c9-4b73-8789-2ae5dccafa87.png">

New `questionAwardedPoints.ejs`:
```
<% include('pointsFormatter'); %>

<% const MAX_DISPLAYED_VARIANTS = 10; %>

<% if (previous_variants) { %>
    <%
        const collapseId = `variants-points-collapse-${instance_question_id}`;
        const collapseButtonId = `variants-points-collapse-button-${instance_question_id}`;
        let startHiddenIndex = null;
        let endHiddenIndex = null;
    %>
    <% if (previous_variants.length > MAX_DISPLAYED_VARIANTS) { %>
        <a id="<%= collapseButtonId %>" role="button" onclick="$(`#<%= collapseId %>`).css('display','inline'); $(`#<%= collapseButtonId %>`).css('display','none');">&hellip;</a>
        <% startHiddenIndex = 0; %>
        <% endHiddenIndex = previous_variants.length - MAX_DISPLAYED_VARIANTS; %>
    <% } %>
    <% for (var i = 0; i < previous_variants.length; i++) { %>
        <% if (i == startHiddenIndex) { %>
            <span id="<%= collapseId %>" style="display:none">
        <% } %>
        <% if (i == endHiddenIndex) { %>
            </span>
        <% } %>
        <span class="badge badge-secondary">
            <a class="text-white" href="<%= urlPrefix %>/instance_question/<%= instance_question_id %>/?variant_id=<%= previous_variants[i].variant_id %>"><%= Math.round(previous_variants[i].max_submission_score * 100) + "%"  %></a> 
        </span>
    <% } %>
<% } %>
```

Reply:
Done in 94d9da247170bc5fee7b100a2a0cd39b3d11bcee